src/fr/charly/javaTutorial/JavaLesson3.java

```diff
@@ -0,0 +1,45 @@
+package fr.charly.javaTutorial;
+
+public class JavaLesson3 {
+
+	public static void main(String[] args) {
+		
+		int money = 2200;
+		
+		if(money !=2000 && money != 2100 && money != 3000) {
+			System.out.println("oui");
+		} else {
+			System.out.println("non");
+		}
+		
+	int age = 11;
```

Comment:
The code is not well indented.
Do "Ctrl + Shift + F" to correct this automatically.